cmd/catalog.go

```diff
@@ -34,7 +34,7 @@ var catalogListCmd = &cobra.Command{
 
 var catalogListComponentCmd = &cobra.Command{
 	Use:   "components",
-	Short: "List all available component types.",
+	Short: "List all components available.",
 	Long:  "List all available component types from OpenShift's Image Builder.",
 	Example: `  # Get the supported components
   odo catalog list components
```

Comment:
Is the word `components` appropriate here as the goal is to list the `builder` images available. I propose/suggest then that you use this command -> `List all builder's images available` and `odo catalog list builders`

Reply:
From user perspective, it is component, right ?